stdlib/Pkg.version

```diff
@@ -1,2 +1,2 @@
 PKG_BRANCH = master
-PKG_SHA1 = 6a235eb813be335b54c97c5c7d631bdbd1059115
+PKG_SHA1 = c2c99807b157d6b9e23d09fa597800ec0e715cee
```

Comment:
Hashes?

Reply:
I'm not sure what you're asking.  :P

Reply:
AH, you're asking for the checksums to be added.  Yes, good call.